lib/webmachine/test/session.rb

```diff
@@ -56,20 +64,51 @@ def do_request(method, uri, options)
         add_query_params(uri, options[:params])
 
         # Set some default headers and merge the provided ones.
-        @headers['Host'] = [uri.host, uri.port].compact.join(':') unless @headers['Host']
-        @headers['Accept'] = '*/*' unless @headers['Accept']
```

Comment:
It was stylistic, not overly important. I use `||=` because it has the same effect, but is more concise. If @headers['Host'] is nil, it will evaluate to false, and thus the assignment will occur. If it has a value, no assignment will occur, just like using unless.

I'm happy to swap it back if you prefer. I think I only made the change because I added another default header using ||=, and just made it consistent.